web-animations/interfaces/Document/getAnimations.html

```diff
@@ -51,5 +51,28 @@
                 'elements in this document');
 }, 'Test document.getAnimations with null target');
 
+async_test(t => {
+  const iframe = document.createElement('iframe');
+  iframe.src = '../../resources/iframe.html';
+  document.body.appendChild(iframe);
+  t.add_cleanup(function() { document.body.removeChild(iframe); });
+  iframe.addEventListener("load", t.step_func_done(function() {
+    const iframeTarget = iframe.contentDocument.getElementById("target");
+    const effect = new KeyframeEffect(
+        iframeTarget,
+        [ { backgroundColor: 'green' }, { backgroundColor: 'red' } ],
```

Comment:
(Nit: I'm not sure we actually need any keyframes here. In the interest of keeping this test minimal perhaps we should drop them?)